atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -77,14 +72,8 @@ def __init__(self, tasker, workflow,
         self.compose_ids = compose_ids
 
         self.koji_target = koji_target
-        self.koji_fallback = {
-            'hub_url': koji_hub,
-            'auth': {
-                'ssl_certs_dir': koji_ssl_certs_dir
-            }
-        }
         if koji_target:
-            if not get_koji(self.workflow, self.koji_fallback)['hub_url']:
+            if not get_koji(self.workflow).get('hub_url'):
```

Comment:
`hub_url` is a required attribute of the `koji` Json Object. Shouldn't this be `['hub_url']` instead?

Reply:
My mistake, I'll fix that

Reply:
Fixed